test/cli/data/gapic/api/artman_pubsub.yaml

```diff
@@ -17,37 +17,56 @@ common:
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/pubsub/v1/pubsub_gapic.yaml
   output_dir: ${REPOROOT}/artman/output
-  git_repo:
-    # FIXME: Change into the main repo after done with testing.
-    location: git@github.com:lukesneeringer/api-client-staging.git
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
   enable_batch_generation: True
-java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-pubsub
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-pubvub-v1
-python:
-  gapic_code_dir: ${REPOROOT}/artman/output/gapic-google-cloud-pubsub-v1
-  git_repo:
-    gapic_subpath: generated/python/gapic-google-cloud-pubsub-v1
-    grpc_subpath: generated/python/proto-google-cloud-pubsub-v1
-go:
-  gapic_code_dir: ${REPOROOT}/gapi-pubsub-go
-  git_repo:
-    gapic_subpath: generated/go/google-pubsub-v1
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-pubsub
+go:
+  gapic_code_dir: ${REPOROOT}/gapi-pubsub-go
+  git_repos:
+    go:
+      gapic_subpath: pubsub/apiv1
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
+    staging:
+      gapic_subpath: google-pubsub-v1/cloud.google.com/go/pubsub/apiv1
+java:
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-pubsub
+  git_repos:
+    java:
```

Comment:
Are you referring to a case where someone wants to generate a single GAPIC in multiple languages _and stage them to the google-cloud-{lang} repos outright_?

My gut tells me this is not something we want to encourage in the long term. (We want CI from api-client-staging to have that role ultimately. I view the `--target-repo` feature as being primarily necessary not to break the workflow in the meantime.)

That said, there is value in consistency. `google-cloud` feels wrong to me, but I can not come up with something that feels better.

This is easy to change later (it only requires editing the artman YAMLs, not the code), so we have some room to play around on this.